src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        int index = new Random().nextInt(Color.values().length);
+        return Color.values()[index].toString();
     }
```

Comment:
It's better to use the `name()` method instead of `toString()` to get the String representation of enum constants to avoid issues in case `toString()` is overridden in the enum. Replace `toString()` with `name()` on line 8.